mobile/src/components/ProfileModule/RecruiterProfileDetails/RecruiterProfileDetails.tsx

```diff
@@ -36,6 +36,14 @@ export const RecruiterProfileDetails: FC<Props> = (props) => {
           {profile.companyName}
         </Text>
       </View>
+      <View style={profileDetailsStyles.item}>
+        <Text style={[profileDetailsStyles.title, typography.text]}>
+          {t(`${Namespaces.Profile}:recruiter_company_label`)}
+        </Text>
+        <Text style={[profileDetailsStyles.text, typography.text]}>
+          {profile.city}
+        </Text>
+      </View>
```

Comment:
great that you found the same code in mobile app!
probably `recruiter_company_label` should be replaced with smth else as it's not compatible with `City` item
also, consider cases when `profile.city` is null or undefined